src/api/app/assets/javascripts/webui/application/bento/script.js

```diff
@@ -1,42 +1,42 @@
 $(function() {
-  
+
```

Comment:
Removing this spaces is unrelated, so either remove it or put it in a different commit :wink: